jobs/cancel-stripe-subscription.js

```diff
@@ -0,0 +1,19 @@
+const env = require('../lib/env')
+const stripe = require('stripe')(env.STRIPE_SECRET_KEY)
+const dbs = require('../lib/dbs')
+const upsert = require('../lib/upsert')
+
+module.exports = async function ({ accountId, stripeSubscriptionId }) {
+  const { payments } = await dbs()
+  await stripe.subscriptions.del(
+    stripeSubscriptionId,
+    async (err, confirmation) => {
+      if (err) {
+        throw err
+      }
+      await upsert(payments, accountId, {
+        stripeSubscriptionId: null
+      })
+    }
+  )
```

Comment:
this is causing one of the conflict errors that you see. You pass a callback to the `.del()` method instead of using the promise pattern.

Try this instead

```js
module.exports = async function ({ accountId, stripeSubscriptionId }) {
  const { payments } = await dbs()
  await stripe.subscriptions.del(stripeSubscriptionId)
  await upsert(payments, accountId, {
    stripeSubscriptionId: null
  })
}
```